alvr/server/cpp/platform/linux/EncodePipeline.cpp

```diff
@@ -75,6 +76,12 @@ std::unique_ptr<alvr::EncodePipeline> alvr::EncodePipeline::Create(std::vector<V
   {
     Info("failed to create VAAPI encoder");
   }
+  try {
+    return std::make_unique<alvr::EncodePipelineNvEnc>(input_frames, vk_frame_ctx);
+  } catch (...)
+  {
+    Info("failed to create NvEnc encoder");
```

Comment:
Would be a nice to have, can we change the logic so that it shows positive messages: "using nvenc", "using vaapi" or "using software encoder" instead of errors?

Reply:
I think it's better to create as a sep PR, to provide the info to the dashboard, which encoder is in use now.